egs2/fleurs/asr1/local/data.sh

```diff
@@ -60,4 +61,10 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
     utils/fix_data_dir.sh data/${train_set}
 fi
 
+if [ ${stage} -eq 3 ] && [ ${stop_stage} -ge 3 ]; then
+    log "stage3: Additional data processing - This should only be called after ASR stage 4"
+    # create file of lids for self-conditioning
+    python local/create_lids.py --data_dir ${asr_data_dir}
```

Comment:
Can you add an example of how the lids look like under `data`?

Reply:
Like a file under `data/lid_sample` that contains a few lines?

Reply:
Yes